cowprotocol/watch-tower
# chore: cleanup repo, readme, add cli script

# Description
This PR finishes off feedback from #60.

# Changes

- [x] Comprehensive editorial review and adjustments to `README.md` to reflect current state of the repo 
- [x] Long-awaited addition of `cli` script into `package.json`
- [x] Parameterized the database location (configured via optional `CLI` option)
- [x] Remove all references to Tenderly

## How to test

1. Use latest `pr` `docker` image.
2. Follow instructions in `README.md` for running from a `docker` image.
3. Confirm that the respective chain achieves sync.

## Related Issues

Fixes #60
